src/components/onboarding_comp.js

```diff
@@ -0,0 +1,26 @@
+function Onboarding() {
+  return (
+    <div className="space-y-6">
+      <div className="text-[#6D6AEC] text-[43px] font-bold">
+        Get Started With Carai
+      </div>
+      <ul className="marker:text-[#6D6AEC] list-disc px-6 space-y-6 font-bold text-lg">
```

Comment:
There's a font-size and font-weight difference please note that in this view

Reply:
These were based on the Figma design sizes, should I revert?